### PR TITLE
Create 2015-06-17-shapes-timestamp-highlight

### PR DESCRIPTION
**Pull request by aneda**
@chriddyp is there another way to remove the boarder beside putting "line"'s "width = 0"?

### CODE REVIEW

**Comment by chriddyp** [comment]
@aneda i don't think so besides making the line color transparent.

File: _posts/python/shapes/2015-06-17-shapes-timestamp-highlight [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+data = Data([trace0])
+
+layout = {
+    # to highlight the timestamp we use shapes and create a rectangular
+    'shapes':[
+        #1st highlight during Feb 4 - Feb 6
+        {
+            'type': 'rect',
+            #x-reference is asigned to the x-values
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-timestamp-highlight]: * assigned

File: _posts/python/shapes/2015-06-17-shapes-timestamp-highlight [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            'xref': 'x',
+            #y-reference is asigned to the plot paper [0,1]
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-timestamp-highlight]: * assigned
also, put a space after the comment as per pep-8

File: _posts/python/shapes/2015-06-17-shapes-timestamp-highlight [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+name: Highlighting Timestamp with Rectangle Shapes
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-timestamp-highlight]: Highlighting Time Series Regions with Rectangle Shapes

**Comment by chriddyp** [comment]
besides comments, :dancer:

**Comment by aneda** [comment]
@chriddyp ready!  if I merge it, does the whole folder get merged?

**Comment by chriddyp** [comment]
@aneda yeah, the folder will be made. :dancer:

**Pull request merged**